app/models/cart.rb

```diff
@@ -18,6 +18,10 @@ def count_total
     data.values.sum
   end
 
+  def total_price
+    (data.map { |k, v| Item.find(k.to_i).price * v }).reduce(:+)
+  end
+
```

Comment:
Eh, this is hard since they are written in different contexts. It's good for now.